src/MessagingAttachment/MessagingAttachment.tsx

```diff
@@ -18,8 +18,9 @@ type Props = {
   fileType: AttachmentFileType;
   icon?: React.ReactNode;
   isUpload?: boolean;
-  onClickAttachment: (attachmentID: string) => void;
-  onRemoveAttachment?: (attachmentID: string) => void;
+  onPreviewAttachment?: () => void;
+  onClickDownload: () => void;
+  onRemoveAttachment?: () => void;
```

Comment:
I see that we've nicely made this function no longer take an `attachmentID` either.

I think we'll accordingly want to remove `attachmentID` from this helper as well:

https://github.com/Clever/components/blob/84123ba6fb8a06aa8c4aa71824fcebad9119a14a/src/MessagingAttachment/MessagingAttachment.tsx#L106-L109

```ts
function handleRemoveClick(e, onRemoveAttachment) { 
  e.stopPropagation(); 
  onRemoveAttachment(); 
} 
```